core/src/main/java/org/apache/accumulo/core/util/Halt.java

```diff
@@ -50,10 +51,9 @@ public void run() {
   public static void halt(final int status, Runnable runnable) {
     try {
       // give ourselves a little time to try and do something
-      new Thread() {
+      new Daemon() {
```

Comment:
We should avoid using Hadoop's Daemon. I doubt that is stable public API. Even if it is, we don't want to increase our dependency on Hadoop library code for this. It should be simple to construct the object assigned to a local variable, set it to daemon, and start it in 3 statements, rather than rely on a the Hadoop library.

Reply:
I didn't even notice that it was a Hadoop class.

Reply:
We had one of our own, but I think you may have deleted it elsewhere in this PR (which is a good thing, as we don't really need a dedicated class for that, especially with all your thread factory stuff being consolidated into one place).

Reply:
Resolved in latest commit.